app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -221,7 +221,9 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       return true;
     } else if(($scope.currentTab == "hawkular" && $scope.emsCommonModel.ems_controller == "ems_container") &&
       ($scope.emsCommonModel.emstype) &&
-      ($scope.emsCommonModel.hawkular_hostname != '' && $scope.emsCommonModel.hawkular_api_port)) {
+      ($scope.emsCommonModel.hawkular_hostname != '' && $scope.emsCommonModel.hawkular_api_port) &&
```

Comment:
why are the above in parenthesis together? they are not related to each other like the lines below... I would find it more readable if they where on seprate lines